src/entryPoints/hyperswitch/AcceptInviteHome.res

```diff
@@ -0,0 +1,147 @@
+@react.component
+let make = () => {
+  open APIUtils
+  let updateDetails = useUpdateMethod()
+  let showToast = ToastState.useShowToast()
+  let (showModal, setShowModal) = React.useState(_ => false)
+  let switchMerchantListValue = Recoil.useRecoilValueFromAtom(
+    HyperswitchAtom.switchMerchantListAtom,
+  )
+  let (merchantListValue, setMerchantListValue) = React.useState(_ => [])
+  let (acceptedMerchantId, setAcceptedMerchantId) = React.useState(_ => [])
+
+  React.useEffect1(() => {
+    let filteredSwitchMerchantList = switchMerchantListValue->Array.filter(ele => !ele.is_active)
+    setMerchantListValue(_ => filteredSwitchMerchantList)
+    let arr = Array.make(~length=filteredSwitchMerchantList->Array.length, false)
+    setAcceptedMerchantId(_ => arr)
+    None
+  }, [switchMerchantListValue])
+
+  let acceptInvite = async _ => {
+    try {
+      let url = getURL(~entityName=USERS, ~userType=#ACCEPT_INVITE, ~methodType=Post, ())
+      let merchantIds = acceptedMerchantId->Array.reduceWithIndex([], (acc, ele, index) => {
+        ele
+          ? acc->Array.push(
+              (
+                merchantListValue->Array.get(index)->Option.getOr(SwitchMerchantUtils.defaultValue)
+              ).merchant_id->JSON.Encode.string,
+            )
+          : ()
+        acc
+      })
+
+      let body =
+        [
+          ("merchant_ids", merchantIds->JSON.Encode.array),
+          ("need_dashboard_entry_response", false->JSON.Encode.bool),
+        ]->LogicUtils.getJsonFromArrayOfJson
+      let _ = await updateDetails(url, body, Post, ())
+      showToast(~toastType=ToastSuccess, ~message="Invite Accepted Successfully", ())
+    } catch {
+    | _ => ()
+    }
+    setShowModal(_ => false)
+  }
+
+  switch merchantListValue->Array.length {
```

Comment:
this switch case looks bit wired can we go with  
<UIUtils.RenderIf condition={merchantListValue->Array.length==1}>
<UIUtils.RenderIf condition={merchantListValue->Array.length>1}>

Reply:
Updated